.github/workflows/linter.yml

```diff
@@ -0,0 +1,20 @@
+name: Bookmark CI
+
+on:
+  push:
+    branches-ignore: [master]
+
+jobs:
+  test:
+    runs-on: macOS-latest
```

Comment:
I wonder if running on macOS makes it faster. I was running on Linux and it took forever.

Reply:
Yea I think the macOS-latest VM provided by GitHub Actions has HAXM installed which has support for hardware acceleration